configs/job_script.default.bash

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# ACME job to be analyzed, including paths to simulation data and observations.
+# Change this name and path as needed
+run_config_file="config.run_name_here"
+# no prefix is needed for jobs running on a laptop or desktop computer
+command_prefix=""
+# change this if not submitting this script from the directory
+# containing run_analysis.py
+mpas_analysis_dir="."
```

Comment:
The difference here is that "." is a good default value, since we usually run out of the directory containing `run_analysis.py`.  So I don't want to make the default something else if everyone would change it to the same thing every time they made a copy.